### DISCUSSION TITLE
[Feature suggestions] mailto

### DISCUSSION THREAD

**Discussion by amblerkr** [dubinc/dub#304] [Ideas]
https://mailtolink.me/

mailto:name@domain.com

Mail can be imported as a template.

**Comment by fmerian** [comment]
hi @amblerkr, thanks for the suggestion! 👋 

to clarify, what do you have in mind?

[mailtolin.me](https://mailtolink.me/) is a great tool. you can draft your email and get the `mailto` link. then go to Dub to shorten it.

do you think of another use case?

**Reply by amblerkr** [reply]
That doesn't work, it just takes you to the main page. If you are not a paid member, you will be taken to the main of dub.sh.

Dub URL: https://share.ambler.kr/mail
Origin URL: mailto:contact@ambler.kr

**Reply by fmerian** [reply]
oh that's right! love the idea to be honest. let's create an issue from this discussion, @steven-tey? 😇
